song_translator/track/models.py

```diff
@@ -84,3 +87,15 @@ class Comment(models.Model):
 
     def __str__(self):
         return f"{self.track_id}: {self.message[:10]}"
+
+
+class AuditTable(models.Model):
+    create_time = models.DateTimeField(auto_now=True)
+    table_name = models.CharField(max_length=100)
+    field_name = models.CharField(max_length=100)
+    old_value = models.TextField(max_length=100, blank=True, null=True)
+    new_value = models.TextField(max_length=100)
+    author = models.ForeignKey('track.User', related_name='audit_owner', null=True, on_delete=models.SET_NULL, default=1)
```

Comment:
I'll suggest you to add field 'record_id' which will contain the ID of record you changed